ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -375,6 +376,26 @@ func (r *RayJobReconciler) getSubmitterTemplate(rayJobInstance *rayv1alpha1.RayJ
 		r.Log.Info("User-provided command is used", "command", submitterTemplate.Spec.Containers[0].Command)
 	}
 
+	// Set PYTHONUNBUFFERED=1 for real-time logging, unless user already set it.
```

Comment:
We can reuse `envVarExists`.

https://github.com/ray-project/kuberay/blob/aa1736359b7613a90767d3a79f3e436d1d593d52/ray-operator/controllers/ray/common/pod.go#L670